app/assets/stylesheets/markets.scss

```diff
@@ -0,0 +1,71 @@
+body {
+  background: url(/assets/Destruction.jpg);
+  background-size: cover;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-color: rgba(0, 0, 0, 1);
+}
+
+h6 {
+  color: #B6CA8C;
+  background-color: rgba(0, 0, 0, -100);
+  display: inline;
+}
+
+.eve-time li {
+  background: transparent;
+}
+
+#footer ul{
+  list-style-type: none;
+}
+
+#video {
+  margin-top: 150px;
+}
+
+#notice_wrapper {
+ position: absolute;
+ top: 0;
+ width: 100%;
+ z-index: 999;
+ background-color: rgba(136, 216, 211, 1);
+ .alert, .notice {
+   padding: .9rem 0;
```

Comment:
`.9` should be written with a leading zero as `0.9`<br>Properties should be ordered color, font-size, font-weight, letter-spacing, margin, padding, text-align